streams/src/test/java/org/apache/kafka/streams/integration/AbstractJoinIntegrationTest.java

```diff
@@ -188,6 +195,44 @@ void runTest(final List<List<KeyValueTimestamp<Long, String>>> expectedResult) t
     }
 
 
+    void runTestWithDriver(final List<List<TestRecord<Long, String>>> expectedResult) {
```

Comment:
Added these two methods for the initial conversion of `StreamTableJoinIntegrationTest`.  As the other tests are converted over the other methods in this class needed for running the tests with the embedded broker will be removed.